packages/esm-patient-outcomes-app/src/smsform/send-sms-form.component.tsx

```diff
@@ -157,6 +175,18 @@ const SendSmsForm: React.FC<SendSmsFormProps> = ({
               inputFieldType="text"
               inputFieldPlaceholder={t('smsReceiver', 'SMS Recipient phone number')}
             />
+            <div>
+              <Select
+                id="locale-select"
+                labelText={t('selectLanguage', 'Select Language')}
+                invalid={!!methods.formState.errors.locale}
+                invalidText={methods.formState.errors.locale?.message}
+                {...methods.register('locale')}>
+                {allowedLocales?.map((allowedLocale) => (
+                  <SelectItem key={allowedLocale} value={allowedLocale} text={allowedLocale} />
```

Comment:
Probably useful to look at how we get display names in the [locale-switching component](https://github.com/openmrs/openmrs-esm-core/blob/533e15df65e4dcf0ef7cc4e7225b24353076fd62/packages/apps/esm-primary-navigation-app/src/components/change-language/change-language.modal.tsx#L47)